server/core/src/main/java/dev/slimevr/tracking/trackers/TrackerDataSupport.kt

```diff
@@ -0,0 +1,17 @@
+package dev.slimevr.tracking.trackers
+
+// Note: there is a -1 offset when receiving via UDP, thus ROTATION must be sent as 0 from firmware.
```

Comment:
Why is there offset? 0 is the default state and rotation is the default state. Make it ROTATION(0). If you need OTHER for some reason, make it 255.

Reply:
It was because I wanted the default 0 to be Other, I don't remember why